server/app.js

```diff
@@ -1,10 +1,25 @@
 import express from 'express';
+import bodyParser from 'body-parser';
+import route from './routes/routes';
 
 const app = express();
 const port = process.env.PORT || 3000;
 
-app.get('/', (req, res) => {
-  res.send('Welcome to Politico!');
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.use('/', route);
```

Comment:
By using just `/` here you will have to prepend your all route definitions in route file with `api/v1`, don't you think you could define this mapping differently to prevent that?